core/src/main/resources/hudson/slaves/JNLPLauncher/main.jelly

```diff
@@ -39,22 +39,27 @@ THE SOFTWARE.
                 <p>
                   ${%slaveAgent.cli.run}
                 </p>
-                <pre>curl -sO ${jenkinsURL}jnlpJars/agent.jar
-java -jar <a href="${rootURL}/jnlpJars/agent.jar">agent.jar</a> -jnlpUrl ${jenkinsURL}${it.url}jenkins-agent.jnlp ${it.launcher.getWorkDirOptions(it)}</pre>
+                <j:set var="copy_agent_jar" value="curl -sO ${jenkinsURL}jnlpJars/agent.jar" />
+                <j:set var="copy_java_cmd" value="java -jar agent.jar -jnlpUrl ${jenkinsURL}${it.url}jenkins-agent.jnlp ${it.launcher.getWorkDirOptions(it)}" />
+                <pre>curl -sO ${jenkinsURL}jnlpJars/agent.jar      <l:copyButton message="text copied" text="${copy_agent_jar}" tooltip="Click to copy" />
```

Comment:
```suggestion
                <pre>curl -sO ${jenkinsURL}jnlpJars/agent.jar      <l:copyButton text="${copy_agent_jar}"/>
```

These `message` and `tooltip` add noting and are not localizable either, so please remove (also applies to other occurrences further down).

Reply:
Removed in 2b634cd49370df8608b9e0cf402d9021be89c88e